core/tests/performance_tests/base.py

```diff
@@ -36,9 +37,22 @@ def _record_page_metrics_from_cached_session(self, page_url):
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
     def _record_page_timings_for_url(self, page_url):
-        self.page_metrics = (
-            self.data_fetcher.get_page_timings_for_url(page_url))
+        page_session_metrics = []
+
+        for _ in range(3):
```

Comment:
I get the sense this constant shouldn't be fixed to 3 and should instead be an argument to this function. Consider making it a default argument if you would like to default it, but I do think it should be customizable.

If you choose to default it, please add a constant in this class rather than introducing a magic number like ``3``.